forcetypeassert.go

```diff
@@ -17,7 +17,7 @@ var Analyzer = &analysis.Analyzer{
 	},
 }
 
-const Doc = "forcetypeassert is finds type assertions which did forcely such as below."
+const Doc = "forcetypeassert is finds type assertions which did forcely such as below"
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
exported const Doc should have comment or be unexported